sympy/printing/rust.py

```diff
@@ -562,6 +562,9 @@ def _print_Assignment(self, expr):
             rhs_code = self._print(rhs)
             return self._get_statement("%s = %s" % (lhs_code, rhs_code))
 
+    def _print_sign(self, expr):
+        return "if (%s == 0.0) { 0.0 } else { (%s).signum() }" % (self._print(expr.args[0]))
```

Comment:
The format string expects two inputs because there are two `%s` so it should be like `% (e, e)`.

Reply:
Thanks, fixed